src/main/java/com/softserve/academy/spaced/repetition/service/AccountService.java

```diff
@@ -1,30 +1,93 @@
 package com.softserve.academy.spaced.repetition.service;
 
 import com.softserve.academy.spaced.repetition.domain.Account;
-import com.softserve.academy.spaced.repetition.domain.enums.LearningRegime;
 import com.softserve.academy.spaced.repetition.domain.RememberingLevel;
+import com.softserve.academy.spaced.repetition.domain.enums.LearningRegime;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotAuthorisedUserException;
 
 import java.util.List;
 
+/**
+ * Works with accounts of users
+ */
 public interface AccountService {
+    /**
+     * Updates account
+     *
+     * @param account updated account
```

Comment:
update

Reply:
no
it use fot method context, so it must be updates

Reply:
Okay, write updates)